frontend/crudLocalStorage.js

```diff
@@ -0,0 +1,60 @@
+function getJobs() {
+    const jobs = JSON.parse(localStorage.getItem("jobs"));
+    if (!jobs) {
+        jobs = [];
+    }
+    return jobs;
+}
+function getjob(id) {
+    if (id != 0 && id != null) {
+        const jobs = JSON.parse(localStorage.getItem("jobs"));
+        if (!jobs) {
+            jobs = [];
+            return null;
+        }
+    }
+    jobs.forEach(e => {
```

Comment:
This will result in error as jobs is declared above but inside of the if statement and hence here it will be undefined.